schema/migration-2-0026-20230422.sql

```diff
@@ -0,0 +1,24 @@
+-- Persistent generated migration.
+
+CREATE FUNCTION migrate() RETURNS void AS $$
+DECLARE
+  next_version int ;
+BEGIN
+  SELECT stage_two + 1 INTO next_version FROM schema_version ;
+  IF next_version = 26 THEN
+    EXECUTE 'ALTER TABLE "tx_out" ADD COLUMN "address_id" INT8 NOT NULL' ;
+    EXECUTE 'ALTER TABLE "tx_out" DROP COLUMN "address"' ;
+    EXECUTE 'ALTER TABLE "tx_out" DROP COLUMN "address_raw"' ;
+    EXECUTE 'ALTER TABLE "tx_out" DROP COLUMN "address_has_script"' ;
+    EXECUTE 'ALTER TABLE "tx_out" DROP COLUMN "payment_cred"' ;
+    EXECUTE 'CREATe TABLE "address"("id" SERIAL8  PRIMARY KEY UNIQUE,"address" VARCHAR NOT NULL,"address_raw" BYTEA NOT NULL,"has_script" BOOLEAN NOT NULL,"payment_cred" hash28type NULL)' ;
```

Comment:
Could this table have an additional field inserted (`stake_address_id`)? Useful to filter used addresses contributing towards a stake.
Absence of this field would require querying `tx_out` table and then doing a [distinct] fetch on `address_id` together